File: javascript/62-unique-paths.js
/**
 * @param {number} m
 * @param {number} n
 * @return {number}
 */
 var uniquePaths = function(m, n) {    
    let r = m + n - 2, min = Math.min(m, n)-1, result = 1
    for (let k = 0; k < min; k++) {
        result *= r-k
        result /= k+1
    }
    return result
}

File: golang/62-unique-paths.go
package golang

func uniquePaths(m int, n int) int {
	r, result := m+n-2, 1
	var min int
	if m < n {
		min = m - 1
	} else {
		min = n - 1
	}
	for k := 0; k < min; k++ {
		result = (result * (r - k)) / (k + 1)
	}
	return result
}